{
  "submission_id": "S70",
  "user_id": "U1",
  "problem_id": "P7",
  "timestamp": "2025-09-26T20:24:35Z",
  "status": "Running... (Testcase 1)",
  "language": "python",
  "test_results": [
    {
      "test_case_number": 1,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.04,
      "memory_usage": 7.6484375,
      "actual_output": "",
      "expected_output": "YES",
      "input": "4 12\n2 3 4 9"
    }
  ]
}